httpie/cli/constants.py

```diff
@@ -84,12 +84,13 @@
 }
 PRETTY_STDOUT_TTY_ONLY = object()
 
-
+FORMAT_OPTION_AUTOMATIC = 'automatic'
```

Comment:
It is generic enough to be used later by other format options (like I would do for #1110).

Reply:
I think the default should simply be `null` (parsed as `None`).

Reply:
Sure.